api/api.go

```diff
@@ -217,10 +233,15 @@ func getMeshStatus(api *API) meshStatus {
 	return strippedStatus
 }
 
-func (api *API) alertGroups(w http.ResponseWriter, req *http.Request) {
-	var err error
-	matchers := []*labels.Matcher{}
-	if filter := req.FormValue("filter"); filter != "" {
+func (api *API) alertGroups(w http.ResponseWriter, r *http.Request) {
```

Comment:
They aren't needed, but I figured as long as the route still exists, it doesn't hurt to offer equivalent functionality. Once this route is deleted, all the code goes anyway.